gatsby-plugin-material-ui/src/gatsby-browser.js

```diff
@@ -13,14 +14,7 @@ export const onInitialClientRender = () => {
   }
 };
 
+// Pass through all StlesProvider props
 export const wrapRootElement = ({ element }, pluginOptions) => {
-  if (pluginOptions.stylesProvider) {
-    return (
-      <StylesProvider {...pluginOptions.stylesProvider}>
-        {element}
-      </StylesProvider>
-    );
-  }
-
-  return element;
+  return <StylesProvider {...stylesProviderProps}>{element}</StylesProvider>;
```

Comment:
@oliviertassinari Do we also need to implement the changes on the server side?

Reply:
Client & Server should have the same configuration.

Reply:
Should I do something here?

Reply:
@hupe1980 @oliviertassinari is `wrapRootElement` with `StylesProvider` the only thing we need in `gatsby-ssr.js` now (so it matches `gatsby-browser.js`)?

Reply:
The ServerStyleSheets component must have the same parameters as the StylesProvider in gatsby-browser.js

```javascript
export const wrapRootElement = ({ element, pathname }, pluginOptions) => {
  const sheets = new ServerStyleSheets(pluginOptions.stylesProvider);
  globalLeak.set(pathname, sheets);

  return sheets.collect(element);
};
```